### ISSUE TITLE
Resize handle example, need one, where?

### ISSUE THREAD

**Issue by WriterStat** [taye/interact.js#267]
This shows cool promise!

Where might a person find?

Would be nice, as most folks need a handle to know, resizable.

Thanks!

I see you have the code in the js to do it and that it's been left up to the implementer.

When I say resize handle I mean like the one found at the bottom of a resizable text area in firefox.

My apologies if I didn't phrase this right.

**Comment by WriterStat** [comment]
Here's my thing, I have a text editor in a div, when I use the interact software based on the demos, it drags and resizes the div the editor is in. But overwrites the div somehow, that the editor is in and doesn't allow editing in the editor after resize or drag.

I was thinking if I could limit the extent to the outside portions of the div and just the div, then the drag resize would work, without the editor inside being messed up.

Or possibly by adding a handle to the outside of the div.  For dragging resizing.

Would like to use this software, but unexpected behavior stumped me. =)  So any thoughts, no matter how crazy are appreciated. My apologies if I might have missed something.  Long night. =)

Thank you very much for the cool software.

**Comment by taye** [comment]
Using the `master` version instead of `1.2.4` will most likely solve your problem of not being able to edit text. If that doesn't work, try using the [ignoreFrom setting](http://interactjs.io/api/#Interactable.ignoreFrom) to prevent drag/resize from the text box. You could also use a [custom actionChecker](http://interactjs.io/api/#Interactable.actionChecker) to add extra conditions for actions.

**Comment by WriterStat** [comment]
Hi Taye! Thank you for the note back!  Much appreciated.

Will do and try it.  The ignoreFrom sounds good  Thanks! I may have to create something a bit custom it seems.  The text editor is a content editable div. As a quick fix, my thought was that hovering over it would display a thin full length rectangle on top allowing only dragging from that. Then possibly clicking anywhere else to edit text would then hide that and allow the edit.  With the resizing portion staying the same, but limited to the very edge/6px; 

Thanks again for the great software!

**Comment by WriterStat** [comment]
Thanks very much for the help. Much appreciated.

Tried, this doesn't seem to work for me.  Anything I put inside the draggable area seems to be unavailable for edit.  Thanks again for the thoughts.  As a lark I also tried zindexing them higher, in case they were behind the draggable area, to put them on top of the stack. But no go.

Note: the master version of the zip seems to be the same 1.2.4 version.

Possibly I didn't implement the ignoreFrom correctly is there an example? I did it clean using the drag_resize demo and dropping in a textarea as a test.

For this to allow draggable elements with other types of elements inside it, possibly a drag handle is needed.  So that the other elements aren't effected.

With that said maybe I didn't implement this right.

Thanks for any thoughts, and the great software.

Using the demo if I want to put a title div in it to drag from.  How would I do it?

Or if I put a textarea inside the draggable area, how would I edit it, and still have it dragged with the entire area?

Thanks again very much for the cool code and the help.

Could there be a way to do a drag handle.  Such as a title area, maybe?

I admit js isn't my best language.
